app/views/sp_components/_all_sp_components.html.erb

```diff
@@ -15,11 +15,11 @@
           <td class="govuk-table__cell">
             <%= component.view_component_type(component.vsp) %>
           </td>
-          <td class="govuk-table__cell"><%= link_to 'View component', sp_component_path(component) %></td>
+          <td class="govuk-table__cell"><%= link_to t('components.view_link'), sp_component_path(component) %></td>
         </tr>
       <% end %>
     </tbody>
   </table>
 <% end %>
 
-<%= link_to 'Create new SP component', new_sp_component_path, class: "govuk-button govuk-button--start", role:"button", draggable:"false" %>
+<%= link_to t('components.create_sp_component'), new_sp_component_path, class: "govuk-button govuk-button--start", role:"button", draggable:"false" %>
```

Comment:
see my suggestion below in the locale files, we can then do this, and reduce the duplication of all the text.
```suggestion
<%= link_to t('components.create_component', type: COMPONENT_TYPE::SP), new_sp_component_path, class: "govuk-button govuk-button--start", role:"button", draggable:"false" %>
```